jenkins/config.xml

```diff
@@ -137,7 +137,7 @@
 					</envVars>
 				</org.csanchez.jenkins.plugins.kubernetes.PodTemplate>
 			</templates>
-			<serverUrl>https://172.28.128.4:8443</serverUrl>
+			<serverUrl>https://kubernetes</serverUrl>
```

Comment:
ah good call, yeah just tested and I needed to add the default namespace 

`curl --insecure https://kubernetes.default ` is available